homeUi/src/main/kotlin/com/gravatar/app/homeUi/presentation/home/share/components/ShareHeader.kt

```diff
@@ -33,13 +34,16 @@ import com.gravatar.app.homeUi.R
 import com.gravatar.app.homeUi.presentation.home.components.BlurredHeaderBackground
 import com.gravatar.app.homeUi.presentation.home.components.topbar.TopBarPickerPopup
 import com.gravatar.app.homeUi.presentation.home.profile.header.MENU_BUTTON_SIZE
+import io.github.alexzhirkevich.qrose.rememberQrCodePainter
 
 @Composable
 internal fun ShareHeader(
     avatarUrl: String,
+    vCardQrCodeData: String,
     modifier: Modifier = Modifier,
 ) {
     var topBarMenuVisible by remember { mutableStateOf(false) }
+    val qrcodePainter: Painter = rememberQrCodePainter(vCardQrCodeData)
```

Comment:
❓ Can we round the corners to match the design better?

Reply:
Done!